utils/iio_stresstest.c

```diff
@@ -307,10 +310,11 @@ static void *client_thread(void *data)
 		if (info->verbose == VERYVERBOSE)
 			printf("%2d: Running\n", id);
 
+		buffer_params.idx = info->buffer_size;
```

Comment:
What is the purpose of this?

Reply:
Idk, the original line lookedl ike this: `buffer = iio_device_create_buffer(dev, info->buffer_size, false);`

I just kept it that way.

Reply:
Oh, ok then. I will check it out at some point. But for this PR it's fine.